services/federator/src/Federator/Options.hs

```diff
@@ -60,9 +60,11 @@ instance FromJSON FederationStrategy where
       _ -> fail "invalid FederationStrategy: expected either allowAll or allowedDomains"
 
 -- | Options that persist as runtime settings.
-newtype RunSettings = RunSettings
+data RunSettings = RunSettings
   { -- | Would you like to federate with everyone or only with a select set of other wire-server installations?
-    setFederationStrategy :: FederationStrategy
+    setFederationStrategy :: FederationStrategy,
+    useSystemCAStore :: Maybe Bool,
+    remoteCAStore :: Maybe FilePath
```

Comment:
TODO: we should probably document these configuration values in docs/reference/config-options.md

Reply:
This is done now :+1: